File: Ejercicios/python/media.py
n = int(input("¿Cuántos números vas a introducir? "))

if n <= 0:
    print("El número de valores debe ser positivo.")
else:
    suma = 0.0
    
    print(f"Introduce los {n} números:")
    for i in range(n):
        valor = float(input())
        suma += valor
    
    media = suma / n
    print(f"La media es: {media}")

File: Ejercicios/cplus/media.cpp
#include <iostream>

int main() {
    int n;
    std::cout << "¿Cuántos números vas a introducir? ";
    std::cin >> n;

    if (n <= 0) {
        std::cerr << "El número de valores debe ser positivo.\n";
        return 1;
    }

    double suma = 0.0;
    double valor;

    std::cout << "Introduce los " << n << " números:\n";
    for (int i = 0; i < n; ++i) {
        std::cin >> valor;
        suma += valor;
    }

    double media = suma / static_cast<double>(n);
    std::cout << "La media es: " << media << '\n';

    return 0;
}
